data-plane/config/broker/500-receiver.yaml

```diff
@@ -76,13 +76,19 @@ spec:
             - mountPath: /etc/tracing
               name: config-tracing
               readOnly: true
+            - mountPath: /etc/receiver-secret-volume
```

Comment:
as path or even volume name can we a more qualifying name, for example, "receiver-tls-secret`? That would clarify what kind of secret is this volume and path about